omnibus/cookbooks/firezone/files/default/ctl-commands/force_renewal.rb

```diff
@@ -1,6 +1,34 @@
 # frozen_string_literal: true
 
 require 'mixlib/shellout'
+require 'uri'
+require 'net/http'
+require 'json'
+
+# rubocop:disable Metrics/MethodLength
+def capture
+  telemetry_file = '/var/opt/firezone/cache/telemetry_id'
+  return unless File.exist?(telemetry_file)
+
+  telemetry_id = File.read(telemetry_file)
+
+  return unless telemetry_id
+
+  uri = URI('https://telemetry.firez.one/capture/')
+  data = {
+    api_key: 'phc_ubuPhiqqjMdedpmbWpG2Ak3axqv5eMVhFDNBaXl9UZK',
+    event: 'firezone-ctl force-cert-renewal',
+    properties: {
+      distinct_id: telemetry_id
+    }
+  }
+  return if File.exist?('/var/opt/firezone/.disable_telemetry') || ENV['TELEMETRY_ENABLED'] == 'false'
+
+  Net::HTTP.post(uri, data.to_json, 'Content-Type' => 'application/json')
+rescue StandardError => e
```

Comment:
Unfortunately `Net::HTTP` doesn't have a common ancestor for all possible connectivity errors, so this will have to do for now.